dbmodel.sql

```diff
@@ -26,6 +26,7 @@ ALTER TABLE `player` ADD `player_character` INT(1) UNSIGNED NOT NULL;
 ALTER TABLE `player` ADD `player_bullets` INT(1) UNSIGNED NOT NULL;
 ALTER TABLE `player` ADD `player_activate` TINYINT UNSIGNED NOT NULL;
 ALTER TABLE `player` ADD `player_hp` TINYINT NOT NULL;
+ALTER TABLE `player` ADD `player_general_store` TINYINT NOT NULL;
```

Comment:
maybe a slightly more semantic name would be better, like 'player_autopick_general_store' ?